gpflow/multioutput/conditionals.py

```diff
@@ -463,3 +444,72 @@ def fully_correlated_conditional_repeat(Kmn, Kmm, Knn, f, *, full_cov=False, ful
             addvar = tf.reshape(tf.reduce_sum(tf.square(LTA), axis=1), (R, N, K))  # R x N x K
             fvar = fvar[None, ...] + addvar  # R x N x K
     return fmean, fvar
+
+
+def _mix_latent_gp(W, g_mu, g_var, full_cov, full_output_cov):
+    r"""
+    Takes the mean and variance of an uncorrelated L-dimensional latent GP
+    and returns the mean and the variance of the mixed GP, `f = W \times g`,
```

Comment:
I would just write `f = W g` but also point out that W is a matrix with shape [...]

Reply:
Alright, I'll remove the `\times`. The `W` matrix doesn't have any leading dimensions though.